src/datepicker/datepicker-service.ts

```diff
@@ -43,7 +43,10 @@ export class NgbDatepickerService {
 
   get model$(): Observable<DatepickerViewModel> { return this._model$.pipe(filter(model => model.months.length > 0)); }
 
-  get select$(): Observable<NgbDate> { return this._select$.pipe(filter(date => date !== null)); }
+  get dateSelect$(): Observable<NgbDate> { return this._dateSelect$.pipe(filter(date => date !== null)); }
+
+  /** @deprecated please use `dateSelect$` */
+  get select$(): Observable<NgbDate> { return this.dateSelect$; }
```

Comment:
This should be removed, not deprecated because `NgbDatepickerService` is the internal implementation. I'd say either:
- don't rename `_select$`  and `get select()` at all, because it's internal (I think I prefer this personally)
- or remove this deprecation completely, because it should not be used anywhere anyway